examples/models/yolo12/main.cpp

```diff
@@ -0,0 +1,168 @@
+#include "inference.h"
```

Comment:
@larryliu0820 have we setup examples repo already? I am wondeirng if we should move it there.